conf/messages.en

```diff
@@ -448,9 +448,7 @@ checkYourAnswers.journeyDetails.vehicleRegistrationNumber = Vehicle registration
 checkYourAnswers.sendDeclaration = Now send your declaration
 checkYourAnswers.sendDeclaration.acknowledgement = I understand that:
 checkYourAnswers.amend.sendDeclaration.acknowledgement.1 = I must bring this updated declaration with me
-checkYourAnswers.amend.sendDeclaration.acknowledgement.2 = I must pay for goods I bring into Great Britain for trade or business use
-checkYourAnswers.amend.sendDeclaration.acknowledgement.3 = I will need to show my declaration and purchase receipts if I am stopped by Border Force
-checkYourAnswers.sendDeclaration.acknowledgement.EU.over.thousand = I need to bring proof my goods were produced in the EU
+checkYourAnswers.sendDeclaration.acknowledgement.EU.over.thousand = I need to bring proof of my goods were produced in the EU
```

Comment:
Will Welsh need updating?

Reply:
you should see welsh now

Reply:
the above changed english message has the correct welsh already